vividus-tests/src/main/resources/story/integration/Batch-level variables and meta-filters.story

```diff
@@ -21,3 +21,10 @@ Scenario: North America
 Meta:
   @locale us ca
 Then `<localeBasedVar>` is equal to `na`
+
+Scenario: Create temporary local file for next batch
```

Comment:
the story is used in 3 batches to check batch variables set in the properties, I would like to do not mix the tests, it's better to move this scenario to another story or create a new one